code/game/objects/structures/flora.dm

```diff
@@ -94,13 +99,30 @@
 	death()
 	return ..()
 
-/obj/structure/flora/pottedplant/attackby(obj/item/weapon/W, mob/user)
-	if (W.edge)
-		user.visible_message(span("warning", "[user] cuts down the [src]"))
-		death()
-		return 1
+/obj/structure/flora/pottedplant/attackby(obj/item/W, mob/user)
+	if(do_after(user, 20, act_target = src))
+		if(!stored_item)
+			if(W.w_class <= ITEMSIZE_NORMAL)
+				stored_item = W

```

Comment:
Isn't this chain missing a `user.drop_item(W)` and a `W.forceMove(src)`? Basically, wouldn't the item be left in the user's hands at the moment?